google/cloud/sql/connector/connector.py

```diff
@@ -68,10 +78,11 @@ def connect(instance_connection_string, driver: str, **kwargs):
     # Return a DBAPI connection
 
     loop = _get_loop()
+    keys = _get_keys()
```

Comment:
nit: we only need `_get_keys()` in the 2nd clause of this if statement

Reply:
Done